common/app/routes/challenges/components/video/Lecture.jsx

```diff
@@ -67,6 +67,11 @@ export class Lecture extends React.Component {
 
     const dashedName = 'foo';
 
+    const embedOpts = {
+      width: '853',
+      height: '480'
+    };
```

Comment:
Since this is constant, defined it at the top of the file outside the component. This prevents a new object from being created and passed down for ever render cycle.